openzwave-sys/build.rs

```diff
@@ -1,15 +1,26 @@
 extern crate gcc;
+extern crate pkg_config;
 
 fn main() {
+    let libopenzwave = pkg_config::Config::new()
+        .atleast_version("1.4").probe("libopenzwave").unwrap();
     let mut c = gcc::Config::new();
+    for inc in libopenzwave.include_paths {
+        // workaround for https://github.com/OpenZWave/open-zwave/issues/771
+        let mut parent = inc.clone();
+        parent.pop();
+
+        c.include(inc);
+        c.include(parent);
+    }
+
     c.file("openzwave-c/options.cc")
      .file("openzwave-c/manager.cc")
      .file("openzwave-c/notification.cc")
      .file("openzwave-c/value_classes/value_id.cc")
-     .include("/usr/include/openzwave") // workaround for https://github.com/OpenZWave/open-zwave/issues/771
-     .include("/usr/local/include/openzwave")
      .cpp(true)
-     .flag("-std=c++11") // to iterate with ranges
+     .flag("-std=c++11")  // to iterate with ranges
+     .flag("-pedantic").flag("-Wall").flag("-Wextra")
```

Comment:
OK, I understand this `-pedantic` is triggering your error with the non-standard-but-accepted variadic macro use. I've confused thoughts here: on one case this can make it easier to support both clang and GCC, but on the other case this will prevent us from using things that work in both compilers. Let's keep it for now, with -Werror this gives no harm.

OK for the `-Wall` and `-Wextra` option.